docs/concepts/flow-runners.md

```diff
@@ -200,13 +200,15 @@ The Prefect CLI command `prefect orion kubernetes-manifest` automatically genera
 
 `KubernetesFlowRunner` supports the following settings:
 
-| Attributes | Description |
-| ---- | ---- |
-| image | String specifying the tag of a Docker image to use for the Job. |
-| namespace | String signifying the Kubernetes namespace to use. |
-| labels | Dictionary of labels to add to the Job. |
-| image_pull_policy | The Kubernetes image pull policy to use for Job containers. |
-| restart_policy | The Kubernetes restart policy to use for Jobs. |
-| stream_output | Bool indicating whether to stream output from the subprocess to local standard output. |
+| Attributes | Description                                                                                                            |
```

Comment:
There's no reason to add all this extra spacing into the table, Mkdocs just strips it out, and this formatting makes the source difficult to read. I think it would be better to go back to the original formatting and add the new attributes.

If you find that attributes are wrapping where they shouldn't, we have a "no-wrap" CSS class that you can apply to a span around the text you don't want to wrap: `<span class="no-wrap">image_pull_policy</span>`. In this case, however, I'm not seeing any unexpected wrapping on building the docs.